### PR TITLE
refactor: introduce Directories utility

### PR DESCRIPTION
**Pull request by Benjozork**
## Summary of Changes

* Introduce `Directories` utility to avoid repetitive concatenations and settings fetches

Discord username (if different from GitHub): someperson#4953

### CODE REVIEW

File: src/renderer/utils/Directories.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+const settings = new Store;
+
+export class Directories {
+
+    static inCommunity(targetDir: string): string {
+        return path.join(settings.get('mainSettings.msfsPackagePath') as string, targetDir);
+    }
+
+    static temp(): string {
+        return path.join(os.tmpdir(), 'flybywire_installer');
```
Comment (nistei) [inline src/renderer/utils/Directories.ts:14]:
```suggestion
        return path.join(settings.get('mainSettings.msfsPackagePath') as string, `flybywire_current_install_${(Math.random() * 1000).toFixed(0)}`);
```

File: src/renderer/utils/Directories.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import Store from "electron-store";
+import path from "path";
+import os from "os";
```
Comment (nistei) [inline src/renderer/utils/Directories.ts:3]:
```suggestion
```

**Review by nistei — CHANGES_REQUESTED** [review]
Added fixes from #190

File: src/renderer/utils/Directories.ts (hunk 1 above)
Comment (Benjozork) [reply src/renderer/utils/Directories.ts:14]: Do we wanna always use the Community folder now ?

Comment (nistei) [reply src/renderer/utils/Directories.ts:13]: As discussed in Discord it is faster and no files are kept locked by fragmenter which caused an issue when deleting before.

**Review by nistei — APPROVED** [review]
LGTM

**Pull request merged**